### PR TITLE
Component/section container

### PR DESCRIPTION
**Pull request by andandandandrew**
What?
Added component "SectionContainer" to repo

How?
Codez

Why?
Internal container for grid stuff

Side Effects?
Probably will want more custom tests written when it is pulled into a project.

### CODE REVIEW

File: components/SectionContainer/stories.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { storiesOf } from '@kadira/storybook'
+// import { WithNotes } from '@kadira/storybook-addon-notes'
```
Comment (bbrock25) [inline components/SectionContainer/stories.js]: @andandandandrew did you want to add this back now that we got the dependency resolved?

Comment (bbrock25) [inline components/SectionContainer/stories.js]: (sorry i should have done it while i was in there)

Comment (andandandandrew) [inline components/SectionContainer/stories.js]: yep, ill get it

File: .storybook/config.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ const CenterDecorator = (story) => (
     justifyContent: "center",
     alignItems: "center",
     margin: "0 0",
-    padding: "20",
+    padding: "0 0",
```
Comment (Jon-Barber) [inline .storybook/config.js]: You could save a whole 2 bytes here.

padding: 0

**Pull request merged**